Topic6/src/main/java/org/ignacio/topic6/User.java

```diff
@@ -0,0 +1,43 @@
+package org.ignacio.topic6;
+
+public class User {
+	
+	
+			private String firstName;
+			private String lastName;
+			private String nickName;
+			
+			public User(){
+			
+			}
+			
+			public User(String nickName, String firstName, String lastName){
+				this.setFirstName(firstName);
```

Comment:
Actually instead of call ste method, it is better to directly assign the value to the variable, this.firstName = firstName, for example